net/wrk/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           compilers 1.0
+
+github.setup        wg wrk 4.0.2
+
+categories          net
+platforms           darwin
+license             Apache-2
+maintainers         jlopezri openmaintainer
+description         wrk HTTP benchmarking tool
+long_description    wrk is a modern HTTP benchmarking tool with optional LuaJIT HTTP request generation.
+homepage            https://github.com/wg/wrk
+
+checksums           rmd160  cb6bab30379d5798d9c124ece099fcff631456c9 \
+                    sha256  67722d1066dae9e6bbead42f22065d4a5eb6f060b1b4bcbfb811132c7b06d530
+
+depends_build       port:perl5
```

Comment:
Use "depends_build-append" to not overwrite dependencies from the portgroups being used.